### PR TITLE
xsel: Add page

### PR DESCRIPTION
**Pull request by SShrike**
I've added a page for xsel, with is essentially the same as xclip, just with a different CLI.

Look over it and tell me what you think, there's some wording I wasn't sure of. I was also wondering if I should document the flags for operating on each of the three selections?

### CODE REVIEW

File: pages/linux/xsel.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# xsel
+
+> Selection manipulation tool.
+> For example xsel can copy STDIN to clipboard or print clipboard to STDOUT.
```
Comment (waldyrious) [inline pages/linux/xsel.md]: I'd write the second sentence as:

"For example, xsel can copy STDIN to the clipboard, or print clipboard contents to STDOUT."

File: pages/linux/xsel.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+- Copy output to clipboard:
```
Comment (waldyrious) [inline pages/linux/xsel.md]: Perhaps "Copy output of a command to the clipboard".

File: pages/linux/xsel.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+`echo 123 | xclip -bi`
+
+- Paste clipboard:
```
Comment (waldyrious) [inline pages/linux/xsel.md]: Why not "Paste clipboard into a file"? We could have a separate example where it is piped into another command, if you want to convey the generality of this operation.

**Comment by waldyrious** [comment]
Simple and straightforward, I like it :) As for the flags, I would attempt to include the words corresponding to "i", "o" and "b" in the command descriptions ("c" is already included as "clear").

File: pages/linux/xsel.md (hunk 2 above)
Comment (SShrike) [inline pages/linux/xsel.md]: How about "Copy STDIN to the clipboard."

File: pages/linux/xsel.md (hunk 3 above)
Comment (SShrike) [inline pages/linux/xsel.md]: How about "Print the clipboard to STDOUT"?

I actually might make some more examples to clarify things.

**Comment by SShrike** [comment]
I'll make some edits once I'm home.

**Comment by SShrike** [comment]
I've made an update, I added some more examples and tried to document each selection. I'm not quite sure I'm happy with how I've documented the selections though.

**Comment by waldyrious** [comment]
Unfortunately, I don't think using the description/command format to document the flags is in line with the project's page formatting guidelines. Fortunately, I don't think we need to be as detailed in describing the flags :) we could simply integrate those words into the descriptions of the commands that use them (and try to change the flags across different examples in a pattern that's reasonably discernible). I can give that a try if you'd like.

> I could also update the xclip page in a separate PR, in order to bring them in line with each other as they both do the same thing, so making the pages as similar as possible might be useful.

👍

**Comment by SShrike** [comment]
> I can give that a try if you'd like.

Sorry for taking ages, I forgot about this issue. Sounds good :+1:

**Comment by waldyrious** [comment]
@SShrike I'm on it, will make a proposal soon. Just one clarification: are you sure the example `cat {{file}} | xsel -bo` uses `-bi` and not `-bi`?

**Comment by SShrike** [comment]
Technically it should be `-bi` when reading from STDIN, but It doesn't seem to matter. I'd suggest changing it to `-bi` though.

**Comment by waldyrious** [comment]
Thanks for confirming. Also, is the first example using `xclip` on purpose, or was that an oversight?

**Comment by SShrike** [comment]
Pretty sure that wasn't on purpose, I probably wrote this at 2 AM or something and was half-asleep.

**Comment by SShrike** [comment]
Ping @waldyrious, any updates?

**Comment by waldyrious** [comment]
@SShrike sorry, I've been meanint to look over PRs to this repo but other stuff has been taking priority. I'll finish this now :)

**Comment by waldyrious** [comment]
Unfortunately the notes I had written down are in a different computer I don't have access to at the moment. I'll merge this as-is and will make a separate PR later with proposed changes. Thanks for your patience!

**Pull request merged**